configure.ac

```diff
@@ -610,15 +610,37 @@ AS_IF([test "$GIT" != "false" && test -d .git],
 AC_DEFINE_UNQUOTED(BUILD_VERSION, "$BUILD_VERSION", Build version)
 AC_SUBST(BUILD_VERSION)
 
-# latest_schema_version <schema_dir>
-#  Determine highest RNG version in the given schema directory. Use git if
-#  available to list managed RNGs, in case there are leftover schema files from
-#  an earlier build of a different version. Otherwise, check all RNGs.
+# schema_files <schema-dir>
+#  List all manually edited RNG schemas (as opposed to auto-generated via make)
+#  in the given directory. Use git if available to list managed RNGs, in case
+#  there are leftover schema files from an earlier build of a different
+#  version. Otherwise, check all RNGs.
+schema_files() {
+    local files="$("$GIT" ls-files "$1"/*.rng 2>/dev/null)"
+
+    AS_IF([test x"$files" = x""],
+          [
+              files="$(ls -1 "$1"/*.rng | grep -v
+                       '/(pacemaker|api-result|crm_mon|versions)[^/]*\.rng')"
+          ])
+    echo $files
+}
+
+# latest_schema_version <schema-dir>
+#  Determine highest RNG version in the given schema directory.
 latest_schema_version() {
-    local files=$(git ls-files "$1"/*.rng 2>/dev/null)
-    AS_IF([test x"$files" = x""], [files=$(ls -1 "$1"/*.rng)])
-    echo "$files" | sed -n -e 's/^.*-\([[0-9]][[0-9.]]*\).rng$/\1/p' dnl
-                  | sort -V | tail -1
+    schema_files "$1" | sed -n -e 's/^.*-\([[0-9]][[0-9.]]*\).rng$/\1/p' dnl
+                      | sort -V | tail -1
+}
+
+# schemas_for_make <schema-dir>
+#  Like schema_files, but suitable for use in make variables.
+schemas_for_make() {
+    local file
+
+    for file in $(schema_files "$1"); do
+        AS_ECHO_N(["\$(top_srcdir)/$file "])
+    done
```

Comment:
Or simply `schema_files "$1" | paste -sd ' '`. We can keep this as a one-line convenience function or drop it.

Optionally we could put the `paste` into `schema_files()` itself and then do a `tr ' ' '\n'` in `latest_schema_version()` prior to `sed`.

Reply:
nice, didn't think of that. I'm thinking `| tr '\n' ' '` since we're already using tr elsewhere

Reply:
Ah, now I remember, the important thing here is adding `$(top_srcdir)/`

Reply:
Oh, duh.

Yeah we could use `tr` or `paste` with `sed` to prepend `$(top_srcdir)/`, but at that point it's not markedly simpler than the for loop.